plugins/repository-hdfs/build.gradle

```diff
@@ -154,29 +154,70 @@ if (fixtureSupported) {
 }
 
 boolean secureFixtureSupported = false;
+boolean vagrantSupported = false;
+boolean virtualBoxInstalled = false;
 if (fixtureSupported) {
   // Only do secure fixture support if the regular fixture is supported,
   // and if vagrant is installed. The ignoreExitValue on exec only matters
   // in cases where the command can be found and successfully started. In
   // situations where the vagrant command isn't able to be started at all
   // (it's not installed) then Gradle still throws ExecException.
-  ByteArrayOutputStream pipe = new ByteArrayOutputStream()
   try {
+    ByteArrayOutputStream pipe = new ByteArrayOutputStream()
     ExecResult runResult = exec {
       commandLine 'vagrant', '--version'
       standardOutput pipe
       ignoreExitValue true
     }
     String output = pipe.toString().trim()
     if (runResult.exitValue == 0) {
-      secureFixtureSupported = (output ==~ /Vagrant 1\.(8\.[6-9]|9\.[0-9])+/)
+      vagrantSupported = (output ==~ /Vagrant 1\.(8\.[6-9]|9\.[0-9])+/)
     } else {
       logger.warn("Could not read installed vagrant version:\n" + output)
     }
   } catch (org.gradle.process.internal.ExecException e) {
     logger.warn("Could not find vagrant: " + e.message)
     // Swallow error. Vagrant isn't installed. Leave secure fixture support off.
   }
+
+  // Finding that HOME needs to be set when performing vagrant updates
+  String homeLocation = System.getenv("HOME")
+  if (homeLocation == null) {
```

Comment:
I can change this to throw an exception instead. If vagrant is installed but the environment can't support calling it, we should probably break earlier in the build with clear reasoning for why.